extensions-core/kafka-indexing-service/src/test/java/io/druid/indexing/kafka/KafkaIndexTaskTest.java

```diff
@@ -476,7 +478,7 @@ public void testIncrementalHandOff() throws Exception
     }
     final String baseSequenceName = "sequence0";
     // as soon as any segment has more than one record, incremental publishing should happen
-    maxRowsPerSegment = 1;
+    maxRowsPerSegment = 2;
```

Comment:
@clintropolis would you tell me why this change is needed?

Reply:
This PR also matches the behavior of #6125 which changes a '>' to a '>=' and whose logic was moved [here in this PR](https://github.com/apache/incubator-druid/pull/6129/files/923537d4090a049f267f96266ec706b5a7cfdf19#diff-30359ba3286f9021542c0e2ef7738572R105). So to not modify the test a ton i upped the count (since with new behavior the test scenario would be pushing every row).

I updated the main description of the PR to reflect this, thanks for the reminder, I forgot :+1: